tests/microbenchmarks.py

```diff
@@ -190,11 +216,14 @@ def test_dict_get(test_case):
             dict.get(i)
             dict.get(1)
             dict.get("a")
-    """), timed=True)
+    """),
+        timed=True)
+
 
 def test_dict_set(test_case):
     print("Running", "test_dictionary_set")
-    test_case.runAsJava(adjust("""
+    test_case.runAsJava(
+        adjust("""
```

Comment:
At column 9: (F405) 'adjust' may be undefined, or defined from star imports: utils